prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -200,7 +200,17 @@ presubmits:
       - image: python:3.7
         command:
         - ./v2/test/presubmit.sh
-  
+
+  - name: kubeflow-pipelines-v2-test
+    cluster: build-kubeflow
+    decorate: true
+    run_if_changed: "^(v2/.*)|(v2/test/presubmit-v2-tests.sh)$"
+    spec:
+      containers:
+        - image: gcr.io/google.com/cloudsdktool/cloud-sdk:latest
```

Comment:
Note, some go tests depend on a real MLMD server at localhost:8080. I think we need one more container for the MLMD server, but I've never set up a sidecar in prow tests before. We need to debug this.

Another approach can be running this test in a KFP pipeline.

Reply:
Do we need side car or another KFP pipeline if we added kubectl port-fprward in the script https://github.com/kubeflow/pipelines/pull/5802/commits/57ebac924b3b9256433af1d29c5b1db660d28a75#diff-88dcf2f21baa096c2451b3d3dfbc18ad795f135bc4b260721d259345aebf1b73R35?

Reply:
Ohh, good point! I missed the `kubectl port-forward` command.
Ideally we should test with an empty MLMD db, because some common code path will only run once, but this is good enough for now.